### PR TITLE
Allow PHP 8

### PR DESCRIPTION
**Pull request by ddeboer**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| Documentation   | 
| License         | MIT

#### What's in this PR?

Allow this package to be installed on PHP 8.

#### Checklist

- [ ] Updated CHANGELOG.md to describe BC breaks / deprecations | new feature | bugfix
- [ ] Documentation pull request created (if not simply a bugfix)

### CODE REVIEW

**Review by dbu — APPROVED** [review]
if the ci agrees, this looks good to merge :+1:

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ php:
     - 7.2
     - 7.3
     - 7.4
+    - 8.0snapshot
```
Comment (veewee) [inline .travis.yml:13]: Php-cs-fixer does not suppor PHP8 yet, but you can run composer with `--ignore-platform-req=php`

And add this env var to run php-cs-fixer:
```
PHP_CS_FIXER_IGNORE_ENV: 1
```

**Comment by Nyholm** [comment]
Let's remove php-cs-fixer from composer.json and use a github action or phar instead.

**Comment by ddeboer** [comment]
What about StyleCI: does that already validate against PHPCS?

File: .travis.yml (hunk 1 above)
Comment (ddeboer) [reply .travis.yml:13]: Removed the dep for now, as suggested by @Nyholm.

**Review by Nyholm — APPROVED** [review]
Thank you

**Review by dbu — APPROVED** [review]
looks good :+1:

**Comment by dbu** [comment]
https://github.com/php-http/vcr-plugin/releases/tag/1.2.0

**Pull request merged**